ExposedAccessKeys/lambda_functions/notify_security.py

```diff
@@ -83,3 +100,32 @@ def publish_msg(subject, message):
         print(e)
         print('Could not publish message to SNS topic "{}"'.format(TOPIC_ARN))
         raise e
+
+print("============= Post to Slack ===========")
```

Comment:
This line isn't needed unless you want to push it into the `notify_slack` function. Otherwise this would just run on file load.